### PR TITLE
Introduce react-tabs for the Help Center tabs.

### LINKED ISSUE #216: Implement HelpCenterTabs component
- Implement HelpCenterTabs component

Part 6 of the implementation of pull #203

### PR DESCRIPTION
**Pull request by afercia**
## Summary

This PR can be summarized in the following changelog entry:

* Adds the Help Center ARIA tabs widget component

## Relevant technical choices:

* Introduces `react-tabs`

## Test instructions

This PR can be tested by following these steps:

* use the following code in ContentAnalysis
```
import React from "react";
import styled from "styled-components";

import YoastTabs from "../../Shared/components/YoastTabs";

export const ContentAnalysisContainer = styled.div`
	min-height: 700px;
	padding: 40px;
	background-color: white;
`;

let items = [
	{
		id: "tab1",
		label: "video tutorial",
		content: <p>This is some content for tab 1. <a href="#">focusable element 1</a></p>,
	},
	{
		id: "tab2",
		label: "knowledge base",
		content: <p>This is some content for tab 2. <a href="#">focusable element 2</a></p>,
	},
	{
		id: "tab3",
		label: "email support",
		content: <p>This is some content for tab 3. <a href="#">focusable element 3</a></p>,
	},
];

/**
 * Returns the ContentAnalysis component.
 *
 * @returns {ReactElement} The ContentAnalysis component.
 */
export default function ContentAnalysis() {
	return <ContentAnalysisContainer>
		<h2>Using "react-tabs"</h2>
		<YoastTabs items={ items } />
	</ContentAnalysisContainer>;
}
```

Questions:
- does `view` need to be named `view`? for consistency with the ScoreAssessments, `html` would be better or, maybe, `content` even better
- does it needs to be passed as html object (as implemented in the original PR see #294) or as string? In the second case, I guess it should use `dangerouslySetInnerHTML` ...

Fixes #216

### CODE REVIEW

**Comment by afercia** [comment]
From an a11y perspective, tabs should work this way:
- only one tab is focusable at a time
- once a tab is focused, navigation through tabs should work with arrow keys only
- instead, tabbing from a focused tab should bring directly to the associated tab panel

screen readers should announce the number of tabs and the currently selected one:

screen shot 2017-09-25 at 14 47 21

when tabbing from a tab to its tabpanel, the tabpanel should get focus and screen readers should announce its name, which is given by `aria-labelledby` that targets the associated tab above the tabpanel:

screen shot 2017-09-25 at 14 47 27

This mix of navigation based on the Tab key and the arrow keys is intentional and meant to provide fast navigation through complex UIs. It is often used also in other ARIA widgets.

ARIA widgets are often based on already existing interfaces in our operating systems, and they try to replicate their interaction. See https://github.com/Yoast/wordpress-seo/issues/5172 with some screenshots of the macOS system preferences UI, which uses tabs. On Windows, a typical example is the Task manager.

**Comment by afercia** [comment]
Renamed `view` to `content` as agreed with @IreneStr and @jcomack

**Comment by Xyfi** [comment]
CR :+1:

I made some minor changes.

File: composites/Plugin/Shared/tests/__snapshots__/YoastTabsTest.js.snap [hunk 1]
```diff
@@ -3,84 +3,80 @@
 exports[`the YoastTabs matches the snapshot 1`] = `
 <div
   className="sc-bdVaJa kXyDjN"
+  data-tabs={true}
+  onClick={[Function]}
+  onKeyDown={[Function]}
 >
-  <div
-    className="react-tabs"
```
Comment (IreneStr) [inline composites/Plugin/Shared/tests/__snapshots__/YoastTabsTest.js.snap]: With this commit, the "react-tabs" className has been lost. Is that a problem?

Comment (IreneStr) [reply composites/Plugin/Shared/tests/__snapshots__/YoastTabsTest.js.snap]: Discussed with @Xyfi. Seems not to be a problem.

**Comment by IreneStr** [comment]
Acceptance 🆗

**Pull request merged**